src/conn/conn_cache.c

```diff
@@ -244,6 +244,12 @@ __wt_cache_create(WT_SESSION_IMPL *session, const char *cfg[])
      */
     if (cache->eviction_target >= cache->eviction_trigger)
         WT_RET_MSG(session, EINVAL, "eviction target must be lower than the eviction trigger");
+    if (cache->eviction_dirty_target > cache->eviction_dirty_trigger)
```

Comment:
Would having the target and trigger equal make sense? I can not think of a good reason for them to be equal. For consistency with eviction_target and eviction_trigger, I would prefer returning an error for greater than or equal to.

Reply:
I was wondering if equal would be ok and follow `eviction_target` and `eviction_trigger`. But WT-8030 defines `eviction_dirty_target <= eviction_dirty_trigger`.
I think it would be great for consistency to do `cache->eviction_dirty_target >= cache->eviction_dirty_trigger` unless there is a strong argument behind it.

Reply:
As agreed on the ticket WT-8030, we will check that the target is lower than the trigger.

Reply:
Done in 654552fddbc5d425d495ccbc856f5c3adf9c1448.